README.md

```diff
@@ -12,13 +12,12 @@ This is an open-source project created to enable customers to integrate the Bitm
 
 ## Platform Support
 
-This library requires at least React Native 0.64+ and React 17+ to work properly. The currently supported platforms are:
+This library requires at least React Native 0.64+ and React 17+ to work properly. The **officially supported** platforms are:
 
-- iOS 14.0+
-- tvOS 14.0+
-- Android API Level 21+
-- Android TV API Level 24+
-- Fire TV FireOS 5.0+
+- **iOS/tvOS:** 14.0+
```

Comment:
@rolandkakonyi @123mpozzi what about iPadOS actually? Is that also supported?

Reply:
Still merging this PR as this is for Android, but if iPadOS is also supported, please update the README again :)

Reply:
@christophaigner iPadOS is a marketing name only, we do nothing for or against to support it, but I will add it there.
Commit: https://github.com/bitmovin/bitmovin-player-react-native/commit/ac05c98ca9a0f0f718e541f8398d443eaf8b2220